packages/connect-agreement/src/entities/Signer.ts

```diff
@@ -0,0 +1,38 @@
+import { SubscriptionHandler } from '@aragon/connect-types'
+
+import Signature from '../entities/Signature'
+import { SignerData, IAgreementConnector } from '../types'
+
+export default class Signer {
+  #connector: IAgreementConnector
+
+  readonly id!: string
+  readonly address!: string
+  readonly agreementId!: string
```

Comment:
We could also remove the `!`s here:
```suggestion
  readonly id: string
  readonly address: string
  readonly agreementId: string
```